registry.json

```diff
@@ -25,6 +25,7 @@
   "text": "github:systemjs/plugin-text",
   "ts": "github:frankwallis/plugin-typescript",
   "tsv": "github:wpf500/plugin-tsv",
+  "textures": "npm:textures",
```

Comment:
This section of the registry is for plugins. Can you move this down to the next section at its alphabetical location?